orbstation/modules/cargo/cadavers.dm

```diff
@@ -0,0 +1,128 @@
+// Cadaver crates: crates that each contain one corpse, for "medical purposes".
+/datum/supply_pack/medical/cadaver
+	cost = CARGO_CRATE_VALUE * 5
+	access_view = ACCESS_MEDICAL
+	crate_type = /obj/structure/closet/crate/coffin/stasis
+	crate_name = "medical cadaver crate"
+
+	///Whether a vampire has arrived on the station this round
+	var/vampire_arrival = FALSE
+	///Percent chance for a medical cadaver to be replaced with a vampire
+	var/vampire_chance = 1
+
+/datum/supply_pack/medical/cadaver/generate()
+	. = ..()
+	var/mob/living/carbon/human/H = locate() in .
+	H.death()
+	if(prob(vampire_chance) && !vampire_arrival)
+		vampire_arrival = TRUE
+		qdel(H)
+		var/mob/living/carbon/human/surprise_dracula = new /mob/living/carbon/human/species/vampire(.)
+		//offer control of the vampire to ghosts
+		var/list/mob/dead/observer/candidates = poll_candidates_for_mob("Do you want to play as a vampire fugitive?", ROLE_FUGITIVE, null, 10 SECONDS, surprise_dracula, null)
+		if(LAZYLEN(candidates))
+			var/mob/dead/observer/chosen = pick(candidates)
+			surprise_dracula.key = chosen.key
+			surprise_dracula.mind.special_role = ROLE_FUGITIVE
+			to_chat(surprise_dracula, span_warning("You are a vampire, freshly escaped from imprisonment by NanoTrasen's secret Occultism Department. You've managed to stow away in place of a medical cadaver, bound for the unknown reaches of the Spinward Sector. \
+									You must do whatever it takes to evade capture."))
+			surprise_dracula.log_message("was made into a vampire fugitive by medical cadaver crate", LOG_GAME)
+			return
+	for (var/part in H.internal_organs) //each cadaver comes with a complimentary set of organs, held in stasis
```

Comment:
`for(var/obj/item/organ/part as anything in corpse.internal_organs)`
(assuming you rename H to corpse)